content/docs/validations.md

```diff
@@ -51,7 +51,7 @@ All available properties to define a validation
 | range: string       | A cell or a range of cells affect by the validation rules. Example: Sheet1!A1:A8 or a whole column as Sheet1!E:E                                                                       |
 | type: string        | number \| text \| date \| list \| textLength \| empty \| notEmpty                                                                                                                      |
```

Comment:
On the current [page](https://jspreadsheet.com/docs/validations) here shoued extra option with typo
![image](https://github.com/user-attachments/assets/e36c010e-6c60-4662-ac2d-1e26d9979bbe)